### DISCUSSION TITLE
Update widgets to modern interfaces

### DISCUSSION THREAD

**Discussion by JKeyRandom** [ImpulseAdventure/GUIslice#519] [Ideas]
Any plans to update the widgets with a visual improvement, buttons, sliders...? similar to the lvgl library? I love Guislice but I feel that updating the widgets would be very good.

Is the project still active today?

**Comment by Pconti31** [comment]
@JosekeyRandom If you open up the GUIsliceBuilder you will see seekbars and toggle buttons with various options available to customize them.


Not sure what else you want
![widgets](https://github.com/ImpulseAdventure/GUIslice/assets/44075106/9b2dce05-d2df-4103-af16-ab37f57e5377)

Paul--

**Comment by JKeyRandom** [comment]
I mean a more modern interface like this, just like you can create a custom button with images, it would be nice if it could be done with more elements (Seekbar, progressbar, Slider...)

![simple_slider](https://github.com/ImpulseAdventure/GUIslice/assets/78388189/14480b46-3fe1-4ccc-8b35-eedd4cafdab8)

Here, for example, I have been able to create a personalized buttons with images and it looks very good.

![main](https://github.com/ImpulseAdventure/GUIslice/assets/78388189/b15f27bc-d960-42da-9000-3ca740cd3c8b)

![Settings](https://github.com/ImpulseAdventure/GUIslice/assets/78388189/c5b9fb8c-41ce-42d7-9f38-77c823758ed9)

Antialiasing could also be applied to the elements so that they look better, with the fonts we are already served using the tft_espi library.

Thank you for your response @Pconti31 , I hope I have explained myself.
